cloudcix_primitives/vlanifns.py

```diff
@@ -0,0 +1,245 @@
+
+    status, msg, successful_payloads = run_podnet(disabled, 3050, successful_payloads)
+    if status == False:
+        return status, msg
+
+    return True, messages[1000]
+
+def scrub(
+    vlan: int,
+    ifname: str,
+    namespace: str,
+    config_file=None
+) -> Tuple[bool, str]:
+    """
+    description:
+        Creates a vlan tagged interface. The it moves the interface to the namespace and bring it up
+
+    parameters:
+        vlan:
+            description: VLAN id assigned to the interface.
+            type: integer
+            required: true
+        ifname:
+            description: Interface name.
+            type: string
+            required: true
+        namespace:
+            description: The VRF network namespace identifier, such as 'VRF123'.
+            type: string
+            required: true
+
+    return:
+        description: |
+            A tuple with a boolean flag indicating if the VLAN tagged interface deletion was successful,
+            and the output or error message.
+        type: tuple
+    """
+    # Define message
+    messages = {
+        1100: f'Successfully removed vlanif {ifname}.{vlan} inside namespace {namespace} ',
+        1101: f'vlanif {ifname}.{vlan} does not exist ',
+
+        3121: f'Failed to connect to the enabled PodNet for vlanif_check payload:  ',
+        3122: f'Failed to connect to the enabled PodNet for vlanif_del payload:  ',
+        3123: f'Failed to run vlanif_del payload on the enabled PodNet. Payload exited with status ',
+
+        3151: f'Failed to connect to the disabled PodNet for vlanif_check payload:  ',
+        3152: f'Failed to connect to the disabled PodNet for vlanif_del payload:  ',
+        3153: f'Failed to run vlanif_del payload on the disabled PodNet. Payload exited with status ',
+    }
+
+    # Default config_file if it is None
+    if config_file is None:
+        config_file = '/opt/robot/config.json'
+
+    status, config_data, msg = load_pod_config(config_file)
+    if not status:
+      if config_data['raw'] is None:
+          return False, msg
+      else:
+          return False, msg + "\nJSON dump of raw configuration:\n" + json.dumps(config_data['raw'],
+              indent=2,
+              sort_keys=True)
+    enabled = config_data['processed']['enabled']
+    disabled = config_data['processed']['disabled']
+
+    def run_podnet(podnet_node, prefix, successful_payloads):
+        rcc = SSHCommsWrapper(comms_ssh, podnet_node, 'robot')
+        fmt = PodnetErrorFormatter(
+            config_file,
+            podnet_node,
+            podnet_node == enabled,
+            {'payload_message': 'stdout', 'payload_error': 'stderr'},
+            successful_payloads
+        )
+
+        payloads = {
+                'vlanif_check': f'ip netns exec {namespace} ip link show {ifname}.{vlan}',
+                'vlanif_del':  f'ip netns exec {namespace} ip link del {ifname}.{vlan}'
+        }
+
+
+        ret = rcc.run(payloads['vlanif_check'])
```

Comment:
https://github.com/CloudCIX/primitives/blob/d60c73c4c38b1341cfe2024ff30e1fbb09e77fcc/cloudcix_primitives/ns.py#L305

I think this pattern in the `ns` primitive for determining if a payload is required to be executed will be the one we will follow.  It reduces the need for exiting early with True, if other payloads are later required.